src/Learning/SNBPredictor/SNBPredictorSelectiveNaiveBayes.h

```diff
@@ -51,44 +52,55 @@ class SNBPredictorSelectiveNaiveBayes : public KWPredictorNaiveBayes
 	// Memoire: appartient a l'appele
 	KWPredictorSelectionReport* GetPredictorSelectionReport();
 
-	// Nombre d'attributs a utiliser pendant l'apprentissage
-	int GetTrainingAttributeNumber();
-
-	///////////////////////////////////////////////////////////////////////////////////////////////////
+	//////////////////////////////////////////////////////////////////////////////////////////////////
 	//// Implementation
```

Comment:
C'est peut-être ici qu'il faut ajouter un commentaire d'une vingtaine de ligne donnant un résumé de l'ensemble des classes impliquée dans les algorithmes du SNB

Reply:
To do in #339